src/bootstrap/bootstrap.py

```diff
@@ -178,6 +179,8 @@ def run(args, verbose=False, exception=False, is_bootstrap=False, **kwargs):
     code = ret.wait()
     if code != 0:
         err = "failed to run: " + ' '.join(args)
+        if help_on_error is not None:
+            err = f"{err}\n{help_on_error}"
```

Comment:
Unfortunately we cannot use f-strings, which are Python 3.6+. `x.py` is currently required to be compatible with Python 2.x though there are discussions about bumping it to Python 3 ([Zulip thread](https://rust-lang.zulipchat.com/#narrow/stream/122651-general/topic/x.2Epy.20and.20.2Fusr.2Fbin.2Fenv.20python)).

This caused the rollup failure in #93929.

Reply:
Fixed, thank you.